tests/test_query.py

```diff
@@ -5,7 +5,7 @@
 
 from .graphql_utils import create_resolve_info
 from .models import (
-    Item,
+    Item, OtherItem
```

Comment:
Can you put `OtherItem` in another line with a comma? Like the following:

```py
from .models import (
    Item,
    OtherItem,
)
```

Reply:
```suggestion
    Item,
    OtherItem,
```